webpack/assets/javascripts/react_app/components/hosts/storage/vmware/index.js

```diff
@@ -196,12 +231,14 @@ StorageContainer.defaultProps = {
   initController: noop,
   fetchDatastores: noop,
   fetchStoragePods: noop,
+  changeCluster: noop,
 };
 
-const mapDispatchToProps = state => {
+const mapStateToProps = state => {
```

Comment:
Because it was a typo I believe, mapDispatchToProps is different function. This is mapStateToProps function (see https://redux.js.org/basics/usage-with-react#implementing-container-components)